### ISSUE TITLE
A11y_.NETCore3.0(WPF)_WPFExpenseItStandAlone_ScreenReader: ScreenReader does not read menuitem shortcut as well as ordering properly under "File" menu.

### ISSUE THREAD

**Issue by grubioe** [dotnet/wpf#1460]
Copied from VSO [951708](https://devdiv.visualstudio.com/DevDiv/_queries/edit/951708/)

Check out Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

Environment Details:
Application Name: .NET Core 3.0 (WPF)

.NET Core SDK version:  3.0.100-preview8-013442
Windows Version: Windows10

ScreenReader Version: Narrator, NVDA 2019.1.1

Pre-requisite:

1) Download the latest .NET Core 3 daily build by going to this link.
2) Install Windows (x86 or x64) versions of sdk listed under Master (3.0.x Runtime) by clicking "Installer" link. 

Repro Steps:

1) Run exe file for "ExpenseItDemo" from the build app folder provided.
2) Main Window for "ExpenseItDemo" solution will appear. ​
3) Start ScreenReader and move to menu items under "File" menu.

Actual:

Issue 1:

ScreenReader does not read the shortcut written beside the menuitem "Create Expense Report".

Issue 2:

ScreeenReader does not read the correct ordering of the menu items under "File" menu. It reads "Exit" 3/3.

This issue has been observed both in Narrator and NVDA.

Expected:

Issue 1:

ScreenReader should read the shortcut(Cntrl+Shift+C) written beside the menuitem "Create Expense Report".

Issue 2:

ScreeenReader should read the correct ordering of the menu items under "File" menu as 1/2 , 2/2.

User Impact:

ScreenReader users will not be able to obtain information about the shortcut present for the menu item and also will not understand the  total no of items present under the menu.

Recommendations:

Please refer to below URL which is repository for accessibility bug fixes.

https://microsoft.sharepoint.com/teams/msenable/Pages/Home.aspx

MAS Reference:

MAS 1.3.1

https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Comment by rladuca** [comment]
The second problem is covered by #1467

**Comment by rladuca** [comment]
Problem 1 is an app bug, not a WPF bug.

You just need the correct accelerator key set.  Doing this allows narrator to fully read the key shortcut.

```
<MenuItem Command="local:MainWindow.CreateExpenseReportCommand" AutomationProperties.AcceleratorKey="Control Shift C"/>
```
